cmd/kpod/spec.go

```diff
@@ -92,8 +117,8 @@ func createConfigToOCISpec(config *createConfig) (*spec.Spec, error) {
 	if config.resources.kernelMemory != 0 {
 		g.SetLinuxResourcesMemoryKernel(config.resources.kernelMemory)
 	}
-	if config.resources.memorySwapiness != 0 {
-		g.SetLinuxResourcesMemorySwappiness(config.resources.memorySwapiness)
+	if config.resources.memorySwappiness != -1 {
```

Comment:
would "<0" be better here?  I think it's pretty controlled now, but might be a good hedge long-term.

Reply:
-1 seems to be the special value (Default which indicates whether to use the Swappiness or not. (At least in Docker.)